### DISCUSSION TITLE
Failed to deploy. AWS provider credentials not found

### DISCUSSION THREAD

**Discussion by mlichlundia** [serverless/serverless#10725] [Q&A] [answered]
Hi, all! I was trying to deploy my application, but I get this error.
I`ve filled my aws credentials andit seems all files are ok. If somebody faced this problem pleasee reply

![image](https://user-images.githubusercontent.com/88280435/154847671-e988768e-0096-4359-bba1-f7d570ebd10a.png)

**Comment by pgrzesik** [comment]
Hello @mlichlundia, how did you set up your credentials? Could you check that they're actually working with running some commands with e.g. `awscli`?

**Reply by mlichlundia** [reply]
I used this one <serverless config credentials --provider aws --key <Access Key ID> --secret <Secret Access Key>>.  When I run aws configure I see, that it is filled with my credentials.

**Reply by pgrzesik** [reply]
Did you try running some `awscli` commands to validate that your credentials are actually correct?

**Reply by mlichlundia** [reply]
Yes, I've checked it now again, everything is ok

**Reply by pgrzesik** [reply]
and you still cannot deploy with `serverless deploy` command? Could you please share your `serverless.yml` config file with potential sensitive parts stripped out?

**Reply by mlichlundia** [reply]
Yes, here is link https://github.com/gultyaev/board-api/blob/master/serverless.yml

**Reply by pgrzesik** [reply]
Could you try reconfiguring AWS with `aws configure` and trying out again?

**Reply by mlichlundia** [reply]
It still hasn't worked
![image](https://user-images.githubusercontent.com/88280435/154864246-c2d198d4-c9e1-40c7-ae13-0716731dcc3b.png)

**Reply by pgrzesik** [reply]
Unfortunatelly I'm out of ideas for now, will report back if I come up with another reason as to why it might be failing.

**Reply by mlichlundia** [reply]
I will wait for it, thank you

**Comment by mlichlundia** [comment]
I've tried this code, here's the result
![image](https://user-images.githubusercontent.com/88280435/154976523-2edb09fa-5ace-4575-8df6-77f7afa4e342.png)

**Comment by mlichlundia** [comment, marked as answer]
Managed to make it work by setting environment variables AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
By some reason config isn't being picked by the aws-sdk
